README.md

```diff
@@ -17,12 +17,13 @@ make sure you set the following environment variables, in `.env`
 | MONGO_URI | mongo host URI |
 | AZURE_SA | Azure Storage account name |
 | AZURE_BLOB_CONTAINER | name of the azure storage blob container |
+| AZURE_SHARE_NAME | name of the azure file share |
 | AZURE_DESTINATION_KEY | azure storage account destination key |
 | DB | name of mongo database to backup |
 | MONGO_USERNAME | username for mongodb |
 | MONGO_PASSWORD | password to authenticate against mongodb |
 | MONGO_AUTH_DB | name of mongo authentication database |
 
-
+You should specify either `AZURE_BLOB_CONTAINER` for blob storage or `AZURE_SHARE_NAME` for a file share.
 
 run `./backup_mongo.sh` inside the container
```

Comment:
Assuming you are happy with the entrypoing change then this line is redundant since you'd either invoke with the `run.sh` or in the compose.

Reply:
yes, it can go away.